# Inputs
float64 x_inches
float64 y_inches
float64 heading_degrees
---
# Outputs
bool completed